include/kernel.h

```diff
@@ -533,7 +533,10 @@ enum execution_context_types {
  * @ingroup kernel_apis
  * @{
  */
+typedef void (*thread_user_cb)(const struct k_thread *thread, void *user_data);
```

Comment:
Minor, but, per convention, function pointer typedefs in kernel.h all follow the `some_name_t` convention rather than `some_name_cb` one.

Reply:
OK.